lib/ansible/modules/lineinfile.py

```diff
@@ -123,6 +123,11 @@
     type: bool
     default: no
     version_added: "2.5"
+  encoding:
+    description:
+      - The character set in which the target file is encoded.
+    type: str
+    version_added: "2.19"
```

Comment:
This should probably have a default of `utf-8` to match the previous expectation that was in place when dealing with bytes comparisons.